### ISSUE TITLE
Extract Features for Optical Flow Images

### ISSUE THREAD

**Issue by sudonto** [bryanyzhu/two-stream-pytorch#30]
Hi @bryanyzhu ,
As title said, how is this possible using your pre-trained model? 

Thank you.

**Comment by bryanyzhu** [comment]
Hi, it should be straightforward. 

Change https://github.com/bryanyzhu/two-stream-pytorch/blob/master/models/flow_resnet.py#L146 this function. Right now, we only output `x = self.fc_action(x)`, which is the prediction softmax scores. If you want the features, for example, from 4th convolution layer `x = self.layer4(x)`. You can simply change `x = self.layer4(x)` to `feature = self.layer4(x)`, and output the feature like `return feature`. So now your output is the feature.   

You can directly use this script https://github.com/bryanyzhu/two-stream-pytorch/blob/master/scripts/eval_ucf101_pytorch/temporal_demo.py. The `spatial_prediction = VideoTemporalPrediction(...)` is the feature you want.
